### ISSUE TITLE
Cleanup unused styles

### ISSUE THREAD

**Issue by antony** [6eDesign/svelte-calendar#9]
Including the component in a sapper build (which has unused style warnings) shows a number of unused styles:

```

node_modules/svelte-calendar/src/Components/Datepicker.html
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (107:0)
105: }
106:
107: .weeks-container {
     ^
108:   height: 311px;
109: }
node_modules/svelte-calendar/src/Components/Datepicker.html
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (142:0)
140: }
141:
142: .flex-container {
     ^
143:   padding: 0;
144:   margin: 0;
node_modules/svelte-calendar/src/Components/Datepicker.html
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (151:0)
149: }
150:
151: .flex-item {
     ^
152:   background: tomato;
153:   margin: 5px;
node_modules/svelte-calendar/src/Components/Datepicker.html
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (163:0)
161: }
162:
163: .flex-item:before {
     ^
164:   content:'';
165:   float:left;
node_modules/svelte-calendar/src/Components/Datepicker.html
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (169:0)
167: }
168:
169: .flex-item-inner {
     ^
170:   display: flex;
171:   justify-content: center;
node_modules/svelte-calendar/src/Components/Datepicker.html
Module Warning (from ./node_modules/svelte-loader/index.js):
Unused CSS selector (99:0)
 97: }
 98:
 99: html {
     ^
100:   box-sizing: border-box;
101: }
```

It's worth cleaning these up I think - even if only for my own sanity :D

**Comment by 6eDesign** [comment]
Thanks @antony - this should be fixed in v0.2.9 (publishing now).  Let me know if you see any other warnings or issues.

**Comment by 6eDesign** [comment]
FWIW - if you're updating for this fix you might want to go to 0.2.10 for a small bug/style fix for IE11 users.

**Comment by antony** [comment]
Works a treat - great work :)

IE11? Wow I hate to think how my site works there :laughing:

I'll make the update, cheers!
